src/Symfony/Component/Validator/Resources/translations/validators.lv.xlf

```diff
@@ -444,11 +444,11 @@
             </trans-unit>
             <trans-unit id="114">
                 <source>This value is too short. It should contain at least one word.|This value is too short. It should contain at least {{ min }} words.</source>
-                <target state="needs-translation">This value is too short. It should contain at least one word.|This value is too short. It should contain at least {{ min }} words.</target>
+                <target>Šī vērtība ir pārāk īsa. Tai būtu jābūt vismaz vienu vārdu garai.|Šī vērtība ir pārāk īsa. Tai būtu jābūt ne mazāk kā {{ limit }} vārdus garai.</target>
```

Comment:
```suggestion
                <target>Šī vērtība ir pārāk īsa. Tai būtu jābūt vismaz vienam vārdam.|Šī vērtība ir pārāk īsa. Tai būtu jābūt ne mazāk kā {{ min }} vārdiem.</target>
```

Reply:
@nicolas-grekas both works but i chose this as it is more consequent with other translations

Reply:
ok, then there's still the placeholder: limit doesn't exist, it's either max or min

Reply:
oops, typo from other translation. fixed